code/modules/surgery/organs/ears.dm

```diff
@@ -119,3 +119,26 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 	damage += 40/severity
+
+/obj/item/organ/ears/robot
+	name = "auditory sensors"
+	icon_state = "robotic_ears"
+	desc = "A pair of microphones intended to be installed in an IPC head, that grant the ability to hear."
+	zone = "head"
+	slot = "ears"
+	gender = PLURAL
+	status = ORGAN_ROBOTIC
+
+/obj/item/organ/ears/robot/emp_act(severity)
+	switch(severity)
+		if(1)
+			owner.Jitter(30)
+			owner.Dizzy(30)
+			owner.Knockdown(200)
+			deaf = 30
+			to_chat(owner, "<span class='warning'>Your robotic ears are ringing, uselessly.</span>")
```

Comment:
bit of a weird comma usage

also, don't robotic ears already have an EMP effect?

Reply:
oh, I'm thinking of cybernetic ears, not robotic ones.